pkg/cloud_provider/file/file.go

```diff
@@ -255,8 +265,9 @@ func (manager *gcfsServiceManager) CreateInstance(ctx context.Context, obj *Serv
 		Tier: obj.Tier,
 		FileShares: []*filev1beta1.FileShareConfig{
 			{
-				Name:       obj.Volume.Name,
-				CapacityGb: util.RoundBytesToGb(obj.Volume.SizeBytes),
+				Name:             obj.Volume.Name,
+				CapacityGb:       util.RoundBytesToGb(obj.Volume.SizeBytes),
+				NfsExportOptions: extractNfsShareExportOptions(obj.NfsExportOptions),
```

Comment:
is this even possible - passing `filev1beta1multishare.NfsExportOptions` to `filev1beta1.FileShareConfig`